modules/configuring-hybrid-ovnkubernetes.adoc

```diff
@@ -60,6 +64,128 @@ where:
 `manifests/` directory for your cluster.
 --
 
+. Open the `cluster-network-03-config.yml` file in an editor and configure OVN-Kubernetes with hybrid networking, such as in the following example:
```

Comment:
The 100+ line addition that starts here does not seem to match the description in the PR notes. Please clarify if this addition belongs here and if it is actual new work or just moved from another file, so I can peer review accordingly; also please adjust the description if possible. Or else if this addition is a Git glitch I hope you can fix this, and then I'll peer review a more correct PR. Thanks!